web/styles/app_components.css

```diff
@@ -1040,6 +1040,15 @@ div.overlay {
     }
 }
 
+#empty-list-message-for-list,
+#empty-list-message-for-table {
```

Comment:
`empty-list-message-for-table` sounds weird. `empty-table-message` or `empty-list-message` might be better.